omise/setting.php

```diff
@@ -0,0 +1,86 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+class Setting extends PaymentModule
+{
+    protected $submit_action = 'omise_save_setting';
```

Comment:
@nimid well, for this case just `const` make code more understandable that this field will not be changed by any part of code.
Btw, can you share the idea behind that you chose `protected $submit_action`?